src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -20,7 +20,9 @@
      [react/text {:style    styles/text
                   :on-press (when mailserver-error?
                               #(re-frame/dispatch [:inbox.ui/reconnect-mailserver-pressed]))}
-      (i18n/label label)]]))
+      (if fetching?
+        (str (i18n/label label) " (" fetching? " requests left)")
```

Comment:
if we show this "requests left" part to the end user let's create another label like `"{{label}} ({{fetching?}} requests left)"`